src/backend/web/routes/feeds.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const Feed = require('../../data/feed');
 const { getFeeds } = require('../../utils/storage');
 const { logger } = require('../../utils/logger');
+// const { protect } = require('../authentication');
```

Comment:
I think you can remove this

Reply:
Once David adds in his code protect will be added back in. @humphd didn't know if I should or just comment in for now as a reminder to add it